src/style.css

```diff
@@ -1 +1,26 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
```

Comment:
Avoid using tag names for styling. Instead of styling the `.stars` class along with the `<div>` tag, use only the class selector.